### ISSUE TITLE
Current beta (x64) doesn't work on Windows 7 64bit

### ISSUE THREAD

**Issue by winks** [rust-lang/rust#29434]
```
> & "C:\Program Files\Rust beta 1.5\bin\cargo.exe"
Process didn't exit successfully: `rustc -vV` (exit code: -1073741515)

C:\Program Files\Rust beta 1.5\bin> .\cargo.exe
Process didn't exit successfully: `rustc -vV` (exit code: -1073741515)
C:\Program Files\Rust beta 1.5\bin> .\rustc.exe -vV
C:\Program Files\Rust beta 1.5\bin> .\rustc.exe -v
```

**Comment by retep998** [comment]
This was temporarily broken and a fix was already on its way but beta came out in the middle of the mess.
https://github.com/rust-lang/rust/pull/29403

**Comment by alexcrichton** [comment]
Fixed in #29403 and we'll generate a new beta soon as well
